### ISSUE TITLE
FileNotFound error during photo retrieval

### ISSUE THREAD

**Issue by Pacamania** [IgnoredAmbience/yahoo-group-archiver#105]
Python exited the script with "FileNotFoundError: [Errno 2] No such file or directory:...", even though I've verified the file(photo) is there and viewable on the group page.  Is there a way to retry the retrieval, picking up where it left off, or exluding certain photos or albums if they are recurrently causing errors?

**Comment by Pacamania** [comment]
Tried rerunning the script, ended with same error at same file.  The file exists and I'm able to download it manually, but how do I resume or complete the archiving process?  How can I skip over this file?

**Comment by IgnoredAmbience** [comment]
FileNotFoundError implies a file attempted to be opened on your local system but failed. Unsure where this would have happened without further debug information.

There's been several changes to the code in the last 20 days, including functionality to skip over already downloaded content. Could you try updating and see if it works now?

**Comment by tssmith2002** [comment]
Python 3.8 32bit W7Pro x64  Error archive_files Couldn't access Files functionality for this group.  This error repeats for files, photos, db, links, calendar from the open group vegoilfuel.  Obviously, photos and files are present and downloadable from the website.  This has happen on all the groups that I am trying to archive.  Messages work great but nothing else.

**Comment by IgnoredAmbience** [comment]
@tssmith2002 This implies that the cookie credentials you are providing to the script are incorrect.

**Comment by tssmith2002** [comment]
If it a open group then you shouldn't need the Cookie, right?

**Comment by IgnoredAmbience** [comment]
No, with open groups you'll only get the email archives without authenticating, and nothing else.

**Comment by tssmith2002** [comment]
Thanks, I am having some difficulty in getting the cookies out but that is probably the issue for me.
